packages/block-library/src/navigation/index.php

```diff
@@ -632,6 +637,14 @@ public static function render( $attributes, $content, $block ) {
 		unset( $attributes['rgbTextColor'], $attributes['rgbBackgroundColor'] );
 
 		$inner_blocks = static::get_inner_blocks( $attributes, $block );
+
+		// This sets the `has_submenus` member variable
+		// which is used by other functions in the class
+		// to determine whether or not a navigation has submenus.
+		// This has to be called before any other code that relies
+		// on the has_submenus member variable.
+		static::has_submenus( $inner_blocks );
```

Comment:
we don't have to do this, if it's called once, it sets the flag and then it just returns it. Is there a problem with htat route?

Reply:
Oh yes you're right!